tests/integrations/raftstore/test_prevote.rs

```diff
@@ -130,7 +130,7 @@ fn test_prevote<T: Simulator>(
 
 #[test]
 fn test_prevote_partition_leader_in_majority_detect_in_majority() {
-    let mut cluster = new_server_cluster(0, 5);
+    let mut cluster = new_node_cluster(0, 5);
```

Comment:
Why use node cluster instead?

Reply:
node cluster utilities less CPU, in my environment it's about 1400% instead of 1600% (for server cluster). I think it can make those cases more stable on CI. However, cases with name `test_server_*` are not changed, they still use server cluster.